script/main.py

```diff
@@ -212,6 +212,37 @@ def percentage(conn, gender):
     print('Percentage of women: ' , percentage_of_women, '%')
   else:
     print('You are entering the wrong gender')
+
+
+def average_age(conn, gender):
+  gender_rows = select_all_gender_and_age(conn)    
+  male = 0
+  female = 0
+  sum_of_age_male = 0
+  sum_of_age_female = 0
+  today = date.today().year
+
+  for item in gender_rows:
+    full_date_of_birth = datetime.strptime(item[1], '%Y-%m-%dT%H:%M:%S.%fZ')
+    yaer_of_birth = full_date_of_birth.date().year  
+    age = today - yaer_of_birth
+    if item[0] == 'male':
+      male = male + 1
+      sum_of_age_male = sum_of_age_male + age
+    else:
+      female = female + 1
+      sum_of_age_female = sum_of_age_female + age
+
+  average_age_of_men = int(sum_of_age_male/male)
+  average_age_of_women = int(sum_of_age_female/female)
+  average_age_overall = int((sum_of_age_male + sum_of_age_female)/(male + female))
+
+  if gender == 'male':
+    print('Average age of men: ', average_age_of_men, ' years')
```

Comment:
if you leave it as float, then here you will get lots of digits after the dot. Learn how to print it correctly in python with for example with precision of 2 digits after the dot.